### DISCUSSION TITLE
공모전 관련해서 문의 드립니다1

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#847] [5. Q&A]
제페토 월드 공모전에 참가하려고 하는데 궁금한 점이 있어서 문의 드립니다.
공모전 마감일이 7월 2일인데, 그 전에 월드 제출 및 심사가 완료되어야 하나요?

만약 그렇다면 월드 심사에 2주일 정도 걸린다고 나와 있어서
늦어도 6월 18일까지 작업을 마감하고 월드를 미리 제출해서 심사를 받아야 한다는 뜻인데
그렇지 않고 제출 및 심사 기간을 고려할 필요가 없다면 일정에 여유가 있을 것 같아서 
어떻게 적용되는지 알고 작업 일정을 FIX하고 싶어요.

알려주세요🙏

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 월드 제출만 7월 1일 23시 59분 59초까지 부탁 드립니다.
대신 저희가 공모전 참석 월드임을 알수있어야 하기에 처음에 참가 접수하셨을때 월드명을 명시하시지 않았다면
월드 제목이나 설명 등에 꼭 명시 부탁 드립니다.
감사합니다.
